<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherApplication</class>
 <widget class="QWidget" name="WeatherApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeatherApplication</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../TTKQrc/TTKWeather.qrc">
    <normaloff>:/image/lb_app_icon</normaloff>:/image/lb_app_icon</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>430</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>321</width>
     <height>370</height>
    </rect>
   </property>
  </widget>
  <widget class="WeatherAddItemTableWidget" name="addItemTableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>261</width>
     <height>370</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="windowIcon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/lb_app_icon);</string>
   </property>
  </widget>
  <widget class="QToolButton" name="windowClose">
   <property name="geometry">
    <rect>
     <x>597</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="minimization">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="windowAbout">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>WeatherAddItemTableWidget</class>
   <extends>QListView</extends>
   <header>weatheradditemtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../TTKQrc/TTKWeather.qrc"/>
 </resources>
 <connections/>
</ui>
